### PR TITLE
Fix reference expansions also in docbook 5

### LINKED ISSUE #181: Change to expath-pkg.xml abbrev breaks inbound links
### Where is the problem

https://github.com/eXist-db/documentation/blob/ea57032318fde596d4ad4db82012685bb42fec26/pom.xml#L14

### What is the problem

The generated app's expath-pkg.xml declares the package's `@abbrev` as `exist-documentation`. Apps such as fundocs expect the abbrev for eXist to be `doc`. (See https://github.com/eXist-db/function-documentation/issues/19#issuecomment-365825860.) 

Solutions: (1) Change `artifactId` from `exist-documentation` to `doc`. Or (2) Change references to this app from `{doc}` to `{exist-documentation}`. I prefer the former.

### Please provide the following
*   exist-db version: 4.0.0
*   documentation version: 4.0.0

### PR DESCRIPTION
**Pull request by duncdrum**
closes #182 
closes #181

### CODE REVIEW

File: src/main/xar-resources/data/contentextraction/listings/listing-3.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 content:get-metadata($binary as xs:base64Binary) as document-node()
 content:get-metadata-and-content($binary as xs:base64Binary) as document-node()
-content:stream-content($binary as xs:base64Binary, $paths as xs:string*, $callback as function, $namespaces as element()?, $userData as item()*) as empty-sequence()
+content:stream-content($binary as xs:base64Binary, $paths as xs:string*, $callback as function, $namespaces as element()?, $userData as item()*) as empty()
```
Comment (dizzzz) [inline src/main/xar-resources/data/contentextraction/listings/listing-3.txt]: `empty()` should be `empty-sequence()`

Comment (duncdrum) [reply src/main/xar-resources/data/contentextraction/listings/listing-3.txt]: indeed something is foul in the state of denmark

Comment (dizzzz) [reply src/main/xar-resources/data/contentextraction/listings/listing-3.txt]: closing the OR? 😱😳

Comment (duncdrum) [reply src/main/xar-resources/data/contentextraction/listings/listing-3.txt]: yes the big one will have to wait another round

**Pull request closed without merging**